artifacts/docker-compose-all.yaml

```diff
@@ -0,0 +1,127 @@
+version: '3.1'
+
+services:
+
+  mongo:
+    network_mode: host
+    image: mongo
+    restart: on-failure
+    ports:
+      - 27017:27017
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: example
```

Comment:
do we want to add those environments to .env file?
environment:
      - MONGO_INITDB_ROOT_USERNAME=${MONGO_INITDB_ROOT_USERNAME}
      - MONGO_INITDB_ROOT_PASSWORD=${MONGO_INITDB_ROOT_PASSWORD}